xml/installation-installation-configure_lbaas.xml

```diff
@@ -724,338 +606,413 @@ rtt min/avg/max/mdev = 3.473/3.473/3.473/0.000 ms</screen>
   <procedure>
    <step>
     <para>
-     Create load balancer for subnet
+     Create the load balancer on the tenant network
```

Comment:
```suggestion
     Create the load balancer on the tenet network:

Reply:
Is it _tenant_ or _tenet_? The rest of the doc talks about _tenant_.

Reply:
tenant 

A group of users; used to isolate access to Compute resources. An alternative term for a project.

Reply:
tenant is correct in this case, I did add the colon at the end though